cli/commands/common.go

```diff
@@ -18,10 +18,15 @@ import (
 
 func getClient(ctx *cli.Context) (*client.Padl, error) {
 	config, err := config.GetConfig(ctx.GlobalString("config"))
+	jwt := ctx.GlobalString("service-account-token")
```

Comment:
just call it 'auth-token' that way everyone can use it, not just service accounts